From 3d23f151b8caf5c17b94bb62e8a9ca29a71ea580 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <balbi@ti.com>
Date: Thu, 13 Oct 2011 11:34:57 +0300
Subject: [PATCH 0358/1092] cbus: tahvo: drop static global pointer

that's not used anywhere anymore. We can
safely drop it.

Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 drivers/cbus/tahvo.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/cbus/tahvo.c b/drivers/cbus/tahvo.c
index 784431c..43f6d6a 100644
--- a/drivers/cbus/tahvo.c
+++ b/drivers/cbus/tahvo.c
@@ -56,8 +56,6 @@ struct tahvo {
 	unsigned int	is_betty:1;
 };
 
-static struct tahvo *the_tahvo;
-
 /**
  * __tahvo_read_reg - Reads a value from a register in Tahvo
  * @tahvo: pointer to tahvo structure
@@ -334,7 +332,6 @@ static int __devinit tahvo_probe(struct platform_device *pdev)
 	tahvo->irq_end	= ret + MAX_TAHVO_IRQ_HANDLERS;
 	tahvo->dev	= &pdev->dev;
 	tahvo->irq	= irq;
-	the_tahvo	= tahvo;
 
 	tahvo_irq_init(tahvo);
 
@@ -388,7 +385,6 @@ static int __devexit tahvo_remove(struct platform_device *pdev)
 	free_irq(irq, 0);
 	irq_free_descs(tahvo->irq_base, MAX_TAHVO_IRQ_HANDLERS);
 	kfree(tahvo);
-	the_tahvo = NULL;
 
 	return 0;
 }
-- 
1.7.11.2

